docs/UserGuide.md

```diff
@@ -251,8 +254,8 @@ Allows you to edit an existing patient in the system. This is useful to correct
 * Existing values will be updated to the input values.
 
 **Examples**:
-*  `edit_p 1 n\Joe Ng` Edits the name of 1st patient to be `Joe Ng`.
-*  `edit_p 2 p\91234567` Edits the phone of the 2nd patient to be `91234567`.
+*  `edit_p 1 n\Joe Ng` edits the name of 1st patient to be `Joe Ng`.
```

Comment:
'1st patient in the patient list' would be clearer, can follow the format in delete_p (line 311)

Reply:
speaking of which, why never add when it is your turn to handle this entire section?

Reply:
so soz missed it out haha just realised